### DISCUSSION TITLE
Playtime: Built-in browser for audio or MIDI clips

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1014] [Ideas]
- Maybe integrate with Pot or even make available in Pot only
- Could use REAPER MediaDBs

**Comment by joungle123** [comment]
@helgoboss I would like to add to this - Integrate plugin browser as in Ableton (for Playtime 2). Browser being on the left side and being able to search (with ctrl+F shortcut) and drag and drop plugins into columns/tracks. This feature would be fantastic and amazing! And of course integrating Sound Pot preset browser possibly as a separate column, so that for example: when you search for plugin, select it then on the other column all presets are visible so we can jump quickly between instruments and presets quickly. The instruments and effect plugins would be really nice to be drag and drop possible. For presets - it could be just double-click and drag and drop as well, so users could choose which option inspires them most.

**Comment by radjoulbalma** [comment]
I had a similar idea docking left the media explorer and browsing with a controler and copying a choosen media in the selected track
